### PR TITLE
(Feature) Decimals on orderbook size bug fix

### LINKED ISSUE #90: Orderbook sizes show more decimals
On the orderbook, the trade size amount shows only two decimals, the rest are zero paddings, even when the real value is not that, for example:

size value: 5.015, UI displays: 5.010

### PR DESCRIPTION
**Pull request by Agupane**
Fix bug that displays only two decimals on the orderbook size
closes #90

### CODE REVIEW

**Comment by mariano-aguero** [comment]
@Agupane LGTM, could you add some tests for this new parameter?

**Comment by Agupane** [comment]
@mariano-aguero it's done

File: src/components/marketplace/order_book.tsx [hunk 1]
```diff
@@ -36,7 +36,7 @@ const orderToRow = (
     selectedToken: Token,
     mySizeOrders: OrderBookItem[] = [],
 ) => {
-    const size = tokenAmountInUnits(order.size, selectedToken.decimals);
+    const size = tokenAmountInUnits(order.size, selectedToken.decimals, 7);
```
Comment (unjapones) [inline src/components/marketplace/order_book.tsx]: I think we should decide how may decimals the UI is going to display (and hide, implicitly) and create the corresponding constants to replace the magic numbers in the functions `padRightSplitted` and `tokenAmountInUnits`.

Based on "Main Market page" mockup, the Order Book may show:
* 4 decimals for "ORDER SIZE" and "MY SIZE" columns.
* 7 decimals for "PRICE (ETH)" column.

Comment (unjapones) [reply src/components/marketplace/order_book.tsx]: Also, just in case: apply the fix to "MY SIZE" column's values.

Comment (Agupane) [reply src/components/marketplace/order_book.tsx]: It's done

Comment (Agupane) [reply src/components/marketplace/order_book.tsx]: it's done

**Comment by mariano-aguero** [comment]
@Agupane One test is failing, can you check it ? Thanks

**Comment by Agupane** [comment]
@mariano-aguero now it's fixed

**Pull request merged**